### DISCUSSION TITLE
o1 model with image failed due to gpt-4* vision model doesn't support max_completion_tokens

### DISCUSSION THREAD

**Discussion by cloud-25683** [danny-avila/LibreChat#4564] [Troubleshooting]
### What happened?

Due to o1 model doesn't support chat with image, my default vision model is gpt-4o, but due to implementation of this [PR](https://github.com/danny-avila/LibreChat/pull/4376), if incoming request is o1 models, max_tokens is converted to max_completion_tokens and max_completion_tokens is not supported by gpt-4o model, so it failed. I understand we are not supposed to use o1 model with image, but I recommend better error handling/message on the UI.

### Steps to Reproduce

1. choose azure openai o1 model
2. upload an image
3. ask a question and UI shows errors: Something went wrong. Here's the specific error message we encountered: An error occurred while processing your request. Please contact the Admin.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
I would fix this, but then o1 would never be used in the conversation, as it would switch to the vision model every time unless you disable "resend files" from "Parameters". Since it's a one line fix, I will do this, but just note this behavior.
